src/service_aggregator.py

```diff
@@ -133,16 +143,37 @@ async def post_async(host_url, query, guid, params=None):
                 # wait the for the message
                 async for message in queue_iter:
                     async with message.process():
-                        response = Response()
-                        response.status_code = 200
-                        response._content = message.body
+                        # build the path/file name
+                        file_name = f'{queue_file_dir}/{guid}-async-data.json'
+
+                        # check to insure file exists
+                        if os.path.exists(file_name):
+                            # open and save the file saved from the callback
+                            with open(file_name, 'r') as f:
+                                # save the data in the file to a Response object
+                                response._content = bytes(f.read(), 'utf-8')
+
+                                # check to insure we got data from the file
+                                if len(response._content):
+                                    # set the status to indicate success
+                                    response.status_code = 200
```

Comment:
hi @PhillipsOwen , these seem to be 500 level errors (?),  client that made a post request to Aragorn initially, is unlikely to cause this issues. But rather some failures in the server's env might be broken (?). Maybe we can report as some 500 codes to the client instead